fs/fs.go

```diff
@@ -470,12 +474,12 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 	select {
 	case l = <-resultChan:
 	case err := <-errChan:
-		log.G(ctx).WithError(err).Debug("failed to resolve layer")
-		retErr = fmt.Errorf("failed to resolve layer: %w", err)
+		log.G(ctx).WithError(err).Debug("unable to resolve layer for lazy loading")
+		retErr = err
 		return
 	case <-time.After(fs.mountTimeout):
-		log.G(ctx).Debug("failed to resolve layer (timeout)")
-		retErr = fmt.Errorf("failed to resolve layer (timeout)")
+		log.G(ctx).WithField("timeout", fs.mountTimeout.String()).Info("timed out waiting for layer to resolve")
+		retErr = fmt.Errorf("timed out waiting for layer to resolve")
```

Comment:
will this show which layer cause timeout?

Reply:
nit: `timed out` -> `timeout`

Reply:
It will now. :)